### PR TITLE
Sct-16 image/task search

### PR DESCRIPTION
**Pull request by erinz2020**
Added partial search and * as wildcard on task creation page and task filter page.

### CODE REVIEW

**Comment by erinz2020** [comment]
What we can do now on task creation page:
We can do partial search.
       For image 2019-06-08-1-105-0125, we can search this image by only typing "2019" or "-06-" or "012", etc.
 2.   Use * as a wildcard, getting results that match any number of characters in place of the asterisk (*)
       For image 2019-06-08-1-105-0125, we can search this image by:
"2019*125" which means the name starts with 2019, ends with 125
or "*06*08*125" which means the name contains "06", "08" and ends with 125
or "*08*1*125*" which means the name contains "08", "1" and "125".
The same rules apply to task search page.

File: api/controllers/api-images-count.js [hunk 1]
```diff
@@ -46,22 +46,27 @@ module.exports = {
...
+    if(!originalFilenameLower.includes("*")) {      
+      query.originalFilenameLower = {
+        contains: originalFilenameLower
+      };
+    }  
+    else {      
+      query.originalFilenameLower = {        
+         like: originalFilenameLower.replaceAll("*","%")
+      };
+    }
```
Comment (naknomum) [inline api/controllers/api-images-count.js:69]: tested both and they work as desired - nice!

**Review by naknomum — APPROVED** [review]
looks good. query changes seem to do as expected.

File: api/controllers/api-images-count.js (hunk 1 above)
Comment (erinz2020) [reply api/controllers/api-images-count.js:69]: Thank you

**Pull request merged**